### PR TITLE
[yamato] Add 2020.1 editor version to package tests.

### PR DESCRIPTION
**Pull request by surfnerd**
### Proposed change(s)

Now that 2020.1 has branched, we need to add that version to our package tests.

### Useful links (Github issues, JIRA tickets, ML-Agents forum threads etc.)
N/A

### Types of change(s)

- [ ] Bug fix
- [ ] New feature
- [ ] Code refactor
- [ ] Breaking change
- [ ] Documentation update
- [x] Other (please describe)
Testing change.

### Checklist
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added updated the changelog (if applicable)
- [ ] I have added necessary documentation (if applicable)
- [ ] I have updated the migration guide (if applicable)

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -180,8 +180,9 @@ jobs:
       - run:
           name: Install Dependencies
           command: |
-            sudo apt-get install nuget
-            nuget install Grpc.Tools -Version 1.14.1 -OutputDirectory protobuf-definitions/
+            sudo apt-get install nuget=2.12
```
Comment (chriselion) [inline .circleci/config.yml]: Probably need to update the directions in https://github.com/Unity-Technologies/ml-agents/blob/master/protobuf-definitions/README.md#set-up--installation too? Maybe just direct those to here?

File: ml-agents-envs/mlagents_envs/rpc_utils.py [hunk 2]
```diff
@@ -132,7 +132,7 @@ def _process_vector_observation(
     # If there's an Inf (either sign) then the result will be Inf
     # See https://stackoverflow.com/questions/6736590/fast-check-for-nan-in-numpy for background
     # Note that a very large values (larger than sqrt(float_max)) will result in an Inf value here
-    # This is OK though, worst case it results in an unnecessary (but harmless) nan_to_num call.
+    # Raise a Runtime error in the case that NaNs or Infinite values make it into the data.
```
Comment (chriselion) [inline ml-agents-envs/mlagents_envs/rpc_utils.py]: Shouldn't be in this PR?

Comment (surfnerd) [reply ml-agents-envs/mlagents_envs/rpc_utils.py]: correct.

**Comment by chriselion** [comment]
```
markdown-link-check -c markdown-link-check.fast.json com.unity.*********/Third 

ERROR: File not found! Please provide a vaild filename as an argument.
markdown-link-check -c markdown-link-check.fast.json Party 

ERROR: File not found! Please provide a vaild filename as an argument.
markdown-link-check -c markdown-link-check.fast.json Notices.md 

ERROR: File not found! Please provide a vaild filename as an argument.
```
Will try to escape the file names properly, if I can ever figure out what that bash magic is doing.

File: .circleci/config.yml (hunk 1 above)
Comment (surfnerd) [reply .circleci/config.yml]: yeah, I'll fix up the docs when I split the PRs.

**Comment by surfnerd** [comment]
closing in favor of #3506 and a not yet created PR for upgrading gRPC.

**Pull request closed without merging**